templates/zerver/help/view-someones-profile.md

```diff
@@ -18,8 +18,18 @@ streams for which [you can see all subscribers](/help/stream-permissions).
 
 !!! Tip ""
 
-    If you are viewing a list of messages, you can also click on a user's
-    profile picture or name and skip to the last step.
+    You can also click on a user's profile picture or name on a
+    message they sent, and skip to the last step.
+
+{tab|mobile}
+
+1. Tap on a user's profile picture or name.
+
+!!! Tip ""
+
+    If you are viewing a private message, you can also tap the information
```

Comment:
This should be "viewing a private message conversation,", right? We don't have views that show just individual messages.

Reply:
Yes, good catch!